modules/rosa-setting-up-cli.adoc

```diff
@@ -0,0 +1,46 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-get-started-cli.adoc
+
+
+[id="rosa-setting-up-cli_{context}"]
+= Setting up the `rosa` CLI
+
+[role="_abstract"]
+To set up the `rosa` CLI, download the latest release, then configure and initialize `rosa`:
+
+.Procedure
+
+. Download the latest release of `rosa` for your operating system from the link:https://github.com/openshift/rosa/releases/latest[`rosa` releases page on GitHub].
++
+. (Optional) After downloading a release, you can rename the executable file you download to `rosa`, and then add `rosa` to your path.
++
+. After downloading `rosa`, enable bash completion for `rosa`. Bash completion helps to automatically complete commands and suggest options when you press `Tab`. The command generates a bash completion file for `rosa` and sources it to your current shell session.
++
+To configure your bash shell to load `rosa` completions for each session, add the following command to your `bashrc` file (`~/.bashrc` or `~/.profile`).
++
+[source,terminal]
+----
+$ . <(rosa completion)
+----
++
+. Enter the following command to verify that your user now has `cluster-admin` access. A cluster administrator can run this command without errors, but a dedicated administrator cannot.
```

Comment:
Should probably indicate here to run the following command in the Bash terminal so they don't think the command should be entered in their `.bashrc` file as was described in the previous step.

Reply:
Thanks! Done.